modules/collection/programs/beets.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}: let
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkOption mkEnableOption mkPackageOption;
+
+  yaml = pkgs.formats.yaml {};
+  cfg = config.rum.programs.beets;
```

(non-blocking suggestion)

```suggestion
  yaml = pkgs.formats.yaml {};
  
  cfg = config.rum.programs.beets;
```